src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECKCATEGORIES = 'bookstore/src/redux/categories/CHECK_CATEGORIES';
```

Comment:
- Please, kindly restructure this to follow the project requirement and conform to using the right duck's pattern below:
``` 
const CHECKCATEGORIES = 'bookstore/categories/CHECK_CATEGORIES';

```

Reply:
Oki, it is done!